test/configs/dancelight.yml

```diff
@@ -0,0 +1,14 @@
+endpoint: wss://dancelight.tanssi-api.network
+block: ${env.DANCELIGHT_BLOCK_NUMBER}
+mock-signature-host: true
+db: ./tmp/db_mba.sqlite
+
+import-storage:
+    System:
+        Account:
+            - - - 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
```

Comment:
I don't have these accounts, but make sure, if copy/pasted - changed to the correct for dancelight

Reply:
nah they are the same, it's mainly alice that we use for sudo